packages/component-base/src/slot-controller.d.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * @license
+ * Copyright (c) 2021 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import { ReactiveController } from 'lit';
+
+export class SlotController implements ReactiveController {
+  constructor(node: HTMLElement);
```

Comment:
The TS constructor signature doesn't match the JS one.

Reply:
Good catch, fixed.